src/main/res/values-fr/strings.xml

```diff
@@ -9,4 +9,11 @@
 	<string name="loadingIconDescription">Chargement en cours…</string>
 	<string name="upgradingTitle">Mise à jour de votre application</string>
 	<string name="upgradingMessage">Vous passez à la nouvelle version. Veuillez patienter pendant la preparation de vos données.</string>
+
+	<string name="connErrorTitle">Pas de connexion Internet</string>
+	<string name="connErrorMessage">Vous ne semblez pas connecté à Internet. Veuillez vérifier votre connexion et réessayer.</string>
+	<string name="connErrorRetry">RECOMMENCEZ</string>
+
+	<string name="backToExit">Veuillez appuyer à nouveau pour quitter</string>
+	<string name="waitMigration">Veuillez attendre la fin du processus de migration</string>
```

Comment:
@abbyad , I need your French skills again here. Also if you want to check the English strings would be OK, the first 3 were proposed by Nicole but the last 2 were added by me.

Reply:
It's not entirely clear what ["back" button](https://github.com/medic/medic-android/blob/aa7091dc25c7ab2064316158da9255087909508c/src/main/res/values/strings.xml#L63) is here, but some suggestions:

```suggestion
	<string name="connErrorTitle">Aucune connexion Internet</string>
	<string name="connErrorMessage">Vous n'êtes pas bien connecté à internet. Veuillez vérifier votre connexion et réessayer.</string>
	<string name="connErrorRetry">RÉESSAYER</string>

	<string name="backToExit">Veuillez appuyer à nouveau pour quitter</string>
	<string name="waitMigration">Veuillez attendre que la migration de données termine</string>
```